api/auth.py

```diff
@@ -0,0 +1,59 @@
+import re
+
+from rest_framework import authentication, permissions
+from rest_framework import exceptions
+from django.conf import settings
+import jwt
+
+class CognomaAuthentication(authentication.BaseAuthentication):
+    def authenticate(self, request):
```

Comment:
Style: blank line following class definition. [PEP8](https://www.python.org/dev/peps/pep-0008/#blank-lines) I think advocates for two blank lines. Personally, I think 1 is sufficient.